packages/core/src/command/hl7-notification/hl7-notification-webhook-sender.ts

```diff
@@ -4,9 +4,8 @@ export const hl7NotificationSenderParamsSchema = z.object({
   cxId: z.string().uuid(),
   patientId: z.string().uuid(),
   message: z.string(),
-  sourceTimestamp: z.string(),
   messageReceivedTimestamp: z.string(),
-  rawDataFileKey: z.string(),
+  timezone: z.string(),
```

Comment:
we want to minimize + shrink interfaces whenever possible - the smaller / simpler the interface, the easier it is to test, invoke, etc.

Have the hl7 webhook sender lambda infer the timezone, don't pass it through - we should just infer it based on hieName.

Create a similar function to this, but it just takes the hieName instead of an ip.
https://github.com/metriport/metriport/blob/develop/packages/mllp-server/src/utils.ts#L91